symfony/console/3.3/bin/console

```diff
@@ -26,4 +37,4 @@ if ($_SERVER['APP_DEBUG']) {
 
 $kernel = new Kernel($_SERVER['APP_ENV'], $_SERVER['APP_DEBUG']);
```

Comment:
Edge case: if you use `--env=prod` (or any environment), when you get to this spot, `$_SERVER['APP_ENV']` is always `dev`. So, the `bin/console` kernel always loads in dev.

Reply:
fixed!